project/make/.dockernsenter

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This can be removed as https://github.com/docker/libcontainer/pull/390 is now merged. I will test this out soon.